cmd/minikube/cmd/config/util.go

```diff
@@ -88,11 +88,67 @@ func GetClientType() machine.ClientType {
 }
 
 func EnableOrDisableAddon(name string, val string) error {
+
 	enable, err := strconv.ParseBool(val)
 	if err != nil {
 		errors.Wrapf(err, "error attempted to parse enabled/disable value addon %s", name)
 	}
 
+	// allows for additional prompting of information when enabling addons
+	if enable {
+		switch name {
+		case "registry-creds":
+			posResponses := []string{"yes", "y"}
+			negResponses := []string{"no", "n"}
+
+			enableAWSECR := AskForYesNoConfirmation("\nDo you want to enable AWS Elastic Container Registry?", posResponses, negResponses)
+			if enableAWSECR {
+				awsAccessID := AskForStaticValue("-- Enter AWS Access Key ID: ")
+				awsAccessKey := AskForStaticValue("-- Enter AWS Secret Access Key: ")
+				awsRegion := AskForStaticValue("-- Enter AWS Region: ")
+				awsAccount := AskForStaticValue("-- Enter 12 digit AWS Account ID: ")
+
+				cluster.CreateSecret(
```

Comment:
I'm working on some changes to make this easier, will get back to you when I do. I just finished cleaning up the k8s interfaces and implemented the client-go. Will get back soon!